### PR TITLE
Utilize new TimeUntil instead of outdated Timer system

### PR DESCRIPTION
**Pull request by kolexxx**
Replace the outdated system with new TimeUntil. Seems to work fine.

### CODE REVIEW

File: code/rounds/InProgressRound.cs [hunk 1]
```diff
@@ -143,7 +143,7 @@ public override void OnSecond()
                 }
                 else
                 {
-                    RoundEndTime += 1f;
+                    TimeUntilRoundEnds -= 1f;
```
Comment (mzegar) [inline code/rounds/InProgressRound.cs:146]: We should be doing `+= 1f` here.

**Review by mzegar — CHANGES_REQUESTED** [review]
Hmm yeah, I think I noticed what you are talking about, it misses the first second of each round. Maybe it might be worth just to add one second onto each round? or if we can investigate where we lose the first second.

File: code/rounds/InProgressRound.cs (hunk 1 above)
Comment (kolexxx) [reply code/rounds/InProgressRound.cs:146]: yeah, just noticed it as well

**Comment by kolexxx** [comment]
> Hmm yeah, I think I noticed what you are talking about, it misses the first second of each round. Maybe it might be worth just to add one second onto each round? or if we can investigate where we lose the first second.

this is occurs because we floor to the nearest integer instead of ceiling it. With flooring we count 00:00 as second. This should be a simple fix

**Pull request closed without merging**